app/models/loader/orchestrate.rb

```diff
@@ -118,6 +118,67 @@ def load
       store_passwords
 
       store_public_keys
+
+      emit_audit
+    end
+
+    SDID = Audit::SDID
+
+    OPS_MAP = {
+      "INSERT" => "add",
```

Comment:
This seems to be humanizing the names? It's important to have a comment which explains where this is used; and why it's called `OPS_MAP`?

Reply:
It's mapping SQL operations to humanized names for formatting a log message.

Reply:
I guess I need to be more explicit. With these questions I am asking for a comment to be added to the code so that the person who comes along will be able to see the explanation. If the response is only in a PR comment then it will be essentially lost once the PR is merged.